src/main/resources/db/changelog/changes/v1.1.1/changelog-v1.1.1.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <changeSet id="MODLISTS-104" author="mweaver@ebsco.com">
+    <comment>Convert the condition values in the Missing Items canned list to sentence case instead of lower case</comment>
+    <sql>
+      UPDATE list_details
+      SET fql_query = '{"item_status": {"$in": ["Missing", "Aged to lost", "Claimed returned", "Declared lost", "Long missing" ] }}'
+      WHERE id = '605a345f-f456-4ab2-8968-22f49cf1fbb6';
+    </sql>
```

Comment:
I thought about trying to do versioning here, but figured the list is _broken_ and the usual reasons for versioning don't really apply with a canned list in cases like this (we won't need to undo, soft delete, view list history,e tc), so I'm just editing in-place